azure-pipelines/templates/stages/build.yml

```
@@ -79,4 +79,23 @@ stages:
       displayName: Pagkage executables
 
     - publish: $(Build.ArtifactStagingDirectory)/package-$(target)
-      artifact: package-$(target)
\ No newline at end of file
+      artifact: package-$(target)
+
+    - job: PublishScripts
+      dependsOn: Build
+      condition: succeeded()
+      steps:
```

its good practice to specify which vmImage you want to use. I don't actually know what the default value is ^^ See https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#use-a-microsoft-hosted-agent for avaliable images
```suggestion
      condition: succeeded()
      pool:
        vmImage: ?-latest
      steps:
```